Add command to run standalone benchmarks

yamadapc opened the pull request:
Prototyping a way to run benchmarks in one-off fashion against a checkout.

yamadapc commented:
```
rm -rf .tmp ; ts-node ./src/index.ts benchmark --target-dir ../parcel --benchmark Hacker --trace --profile ; open .tmp
```

marcins commented on yarn.lock:
This seems to be references to `yarnpkg.com` changing to `npmjs.org` - is that expected?

marcins commented on src/index.ts:
```diff
@@ -263,7 +275,67 @@ async function start() {
+if (process.env.IS_GH_ACTION === 'true') {
+  start().catch((err) => {
+    console.error(err);
+    process.exit(1);
```
/nit this should be `process.exitCode = 1` to avoid potentially truncating / missing the `console.error` (also above)

MonicaOlejniczak commented on package.json:
```diff
@@ -9,7 +9,7 @@
     "watch": "tsc -w",
     "start": "NODE_ENV=development ts-node src/index",
-    "action-start": "ts-node src/index",
+    "action-start": "IS_GH_ACTION=true ts-node src/index",
```
Is this env meant to be here?

MonicaOlejniczak commented on src/index.ts:
```diff
@@ -263,7 +275,67 @@ async function start() {
+    .parse(process.argv);
+}
+
+if (process.env.IS_GH_ACTION === 'true') {
```
/nit could use a `NODE_ENV` development check provided GH don't provide that

yamadapc replied:
Not sure if there's a difference... `NODE_ENV` I'm not sure should be used because it triggers other changes in behaviour.

There could be a separate entry-point, but then I need to move quite a bit more stuff around;

At the moment I am just keen to get some sort of benchmarking workflow working properly

yamadapc commented:
Thank you @MonicaOlejniczak @marcins; I have addressed most of your comments.

I haven't changed the entry-point part to avoid rewriting this file entirely in this diff. We can split the two parts in another diff.
